src/Cooker/providers/GitProvider/getHashListFromHash.js

```diff
@@ -6,22 +6,30 @@ function getHashListFromHashToCommit(sha, commit) {
       `Missing hash parameter. For commits from origin of repository, use 'Big Bang' as hash.`
     )
   }
+
   return new Promise((resolve, reject) => {
-    let done = false
+    let hitPreviousRelease = false
     const list = []
     commit
       .history()
       .on('commit', commit => {
-        if (done) {
-          // discard
-        } else if (commit.sha() === sha) {
-          done = true
-          resolve(list)
+        if (hitPreviousRelease) {
+          return
+        }
+
+        if (commit.sha() === sha) {
+          hitPreviousRelease = true
         } else {
           list.unshift(commit.sha())
         }
       })
-      .on('end', () => done || resolve(sha === 'Big Bang' ? list : []))
+      .on('end', () => {
```

Comment:
@christianalfoni I think we need to have some way to say "use all commits from the start of the repo". Without `Big Bang` there is no way to do this without skipping initial commit.

As I said, I would not use this artificial 50 limit because it can cause trouble. Either the 'hash' is found or is 'Big Bang' or we throw an error...